src/app/navigation/navigation.component.html

```diff
@@ -0,0 +1,27 @@
+<nav class="navbar navbar-expand-sm bg-dark">
+  <ul class="navbar-nav">
+    <li class="nav-item">
+      <a class="nav-link" routerLink="/" routerLinkActive="active" class="item"
+        >Home</a
+      >
+    </li>
+    <li class="nav-item">
+      <a
+        class="nav-link"
+        routerLink="/products"
+        routerLinkActive="active"
```

Comment:
jestes pewien ze dobrze dziala ci routerLinkActive? i czy jak wchodzisz do menu products to nie zaznacza ci rowniez tego pierwszego (Home) ?

Reply:
Faktycznie nie dziala, naprawie